### DISCUSSION TITLE
New Reporting!

### DISCUSSION THREAD

**Discussion by KenGrant** [amidaware/tacticalrmm#1668] [General]
Loving the fact we now have reporting, but I can't get to grips with it - youtube perhaps?

I have tried to generate a csv report for all clients with some details (Name, Operating system, etc) withoiut a filter (all clients) - tried dozens of times but always seems to need a filter?

We're gonna need a Bigger Lesson!

Please

**Comment by dinger1986** [comment]
Are you on discord? There's loads of tidbits there and some videos. We are planning some videos on YouTube

**Reply by KenGrant** [reply]
I am, so I'll head over there now. Apprecaite all the hard work on this, it seems a shame if it not fully explained how to use it.

Many thanks.

**Reply by dinger1986** [reply]
We pride ourselves on our docs so will look to improve that.

Dm me when you get on discord please.
